website/pages/docs/plugins/sources/homebrew/_configuration.md

```diff
@@ -8,8 +8,8 @@ spec:
   tables: ["*"]
   destinations:
     - "DESTINATION_NAME"
+  backend_options:
+    table_name: "test_state_table"
```

Comment:
Bit of a nit, but can we maybe call it something other than `test_state_table` (here and in the other configs). Maybe `homebrew_state_table`?

Reply:
we have only a single place where we did this, so I left others as is

Reply:
Renamed to `cq_state_<plugin>` in 036cfc5

Reply:
Ah yeah I see, thanks for the fix!